### ISSUE TITLE
Classifier Training Freezes

### ISSUE THREAD

**Issue by lorengaravaglia** [davidsandberg/facenet#917]
Hi All,

I'm having some issues running the classifier training on my own database using inception resnet v1. I followed the wiki for setting up my database. Several times now I've been training the classifier and it just freezes during the batch processing. It's frozen so far during the 20, 30, and 65th epochs at what seems to be a random batch number.  Below are my arguments. 

I wasn't sure if this could be caused by a hardware issue with my GTX 1080. I monitored the temperatures and those seemed to be fine.

I was able to successfully complete the training to 90 epochs once, but I forgot to specify the models base directory and couldn't find the location of the output files. Since then I've had no luck.

Does anyone know what might be causing this?

logs_base_dir: C:/facenet-master/logs/facenet/
models_base_dir: C:/facenet-master/models/facenet/
gpu_memory_fraction: 1.0
pretrained_model: None
data_dir: C:/FaceTrainingDB_mtcnnpy_182/
model_def: models.inception_resnet_v1
max_nrof_epochs: 90
batch_size: 90
image_size: 160
epoch_size: 1000
embedding_size: 512
random_crop: True
random_flip: True
random_rotate: False
use_fixed_image_standardization: True
keep_probability: 0.8
weight_decay: 0.0005
center_loss_factor: 0.0
center_loss_alfa: 0.95
prelogits_norm_loss_factor: 0.0005
prelogits_norm_p: 1.0
prelogits_hist_max: 10.0
optimizer: ADAM
learning_rate: -1.0
learning_rate_decay_epochs: 100
learning_rate_decay_factor: 1.0
moving_average_decay: 0.9999
seed: 666
nrof_preprocess_threads: 4
log_histograms: False
learning_rate_schedule_file: C:/facenet-master/data/learning_rate_schedule_classifier_casia.txt
filter_filename: 
filter_percentile: 100.0
filter_min_nrof_images_per_class: 0
validate_every_n_epochs: 5
validation_set_split_ratio: 0.05
min_nrof_val_images_per_class: 0
lfw_pairs: data/pairs.txt
lfw_dir: 
lfw_batch_size: 100
lfw_nrof_folds: 10
lfw_distance_metric: 1
lfw_use_flipped_images: False
lfw_subtract_mean: False
